infra/terraform/ec2.tf

```diff
@@ -0,0 +1,98 @@
+data "aws_ami" "ubuntu" {
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["${var.ubuntu_ami_name}"]
+  }
+
+  filter {
+    name   = "virtualization-type"
+    values = ["${var.virt_type}"]
+  }
+
+  owners = ["${var.canonical_account}"]
+}
+
+resource "aws_security_group" "owtf_sg" {
+  name   = "owtf_sg"
+  vpc_id = aws_vpc.vpc.id
+
+  ingress {
+    from_port   = var.owtf_admin_port
+    to_port     = var.owtf_admin_port
+    protocol    = "tcp"
+    cidr_blocks = [var.cidr_block]
+  }
+  ingress {
+    from_port   = var.owtf_ui_port
+    to_port     = var.owtf_ui_port
+    protocol    = "tcp"
+    cidr_blocks = [var.cidr_block]
+  }
+  ingress {
+    from_port   = var.owtf_proxy_port
+    to_port     = var.owtf_proxy_port
+    protocol    = "tcp"
+    cidr_blocks = [var.cidr_block]
+  }
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = [var.cidr_block]
+  }
+}
+
+resource "aws_instance" "ec2" {
+  ami                         = data.aws_ami.ubuntu.id
+  instance_type               = var.instance_type
+  associate_public_ip_address = false
+  key_name                    = aws_key_pair.keypair.key_name
```

Comment:
I'd remove the keypair option since the instance is not going to be public. We should recommend the user to use AWS SSM to operate and debug the instance.